index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang='en'>
+  <head>
+    <meta charset='UTF-8'/>
+    <meta name="description"/>
+    <title>Sedona cosik</title>
+  </head>
+
+  <body>
+    <div class="wrapper"><!-- main page -->
+      <header class="page-header">
+        <nav class="main-nav">
+          <ul class="main-nav__list">
+            <li class="main-nav__item main-nav__item--active"><a href="index.html">Home</a></li>
+            <li class="main-nav__item"><a href="photos.html">Photos and videos</a></li>
+            <li class="main-nav__item"><a href="form.html">Feedback Form</a></li>
+            <li class="main-nav__item">HTML Academy</li>
+          </ul>
+          <button class="main-nav__toggle main-nav__toggle--close"></button>
+          <a class="main-nav__link" href="index.html"><img src="" class="main-nav__logo" alt="Sedona logo"></a>
+        </nav>
+
+        <div calss="intro">
+          <ul class="intro-list">
+            <li class="intro-list__item intro-list__item--top">Welcome to the gorgeous</li>
+            <li class="intro-list__item intro-list__item--middle">Sedona</li>
+            <li class="intro-list__item intro-list__item--bottom">"Because the Grand Canyon sucks!"</li>
+          </ul>
+        </div>
+      </header>
```

Comment:
also you have header, I like puting content section into main tag. so that I have something like that:
```
<header></header>
<main></main>
<footer></footer>
```
The idea is that header and footer are common components for all the pages and man section is changing.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main